src/components/Image.tsx

```diff
@@ -0,0 +1,13 @@
+import NextImage, { getImageProps } from 'next/image';
+import { ComponentProps } from 'react';
+
+export default function Image(props: ComponentProps<typeof NextImage>) {
+  const { props: nextProps } = getImageProps({
+    ...props,
+  });
+
+  // eslint-disable-next-line no-unused-vars
+  const { style: _omit, ...delegated } = nextProps;
+
+  return <img {...delegated} />;
+}
```

Comment:
[blocking question] I see we're handling inline styles for images. Are the inline styles for the `ProgressBar` component affected at all by the new CSP headers?

Reply:
I believe we'll be able to grab the nonce off the headers and add it directly to the element using inline styles.

Reply:
As this is a breaking change for the progress bar, could you show how you would add the nonce to the progress bar component in this PR?